code/datums/achievements/antagmastery.dm

```diff
@@ -0,0 +1,112 @@
+/datum/award/achievement/antagmastery
+	category = "Antag Mastery"
+	reward = 400
+
+/datum/award/achievement/antagmastery/on_unlock(mob/user)
+	. = ..()
+	for(var/A in SSachievements.antag_mastery_list)
```

Comment:
This should fail if the list is empty, for cases where DBcore fails to connect or some other error case